lambda_function.py

```diff
@@ -0,0 +1,88 @@
+import json
+import requests
+import os
+
+
+def lambda_handler(event, context):
+    # Base URL of Gupshup's OTP API, including all constant params
+    base_URL = os.environ.get("OTP_API_BASE_URL")
+
+    response_headers = {"Access-Control-Allow-Origin": "*"}
+    event_path = str(event["path"])
+
+    # Check for query params, otherwise return 404 (Parameters not found)
+    if not event["queryStringParameters"]:
+        return {
+            "statusCode": 404,
+            "headers": response_headers,
+            "body": json.dumps("Parameters not found"),
+        }
+
+    query_params = event["queryStringParameters"]
+
+    # Check for 'phone' param, otherwise return 404 (Phone number not found)
+    if "phone" not in query_params:
+        return {
+            "headers": response_headers,
+            "statusCode": 404,
+            "body": json.dumps("Phone number not found"),
+        }
+
+    phone_number = query_params["phone"]
+
+    # Use default values if params aren't provided
+    msg = (
```

Comment:
Why the parantheses in all these variables?

Reply:
I think it's just syntactic sugar. Parentheses are needed because she's also using a ternany if/else comparison

Reply:
It's not needed, and this sugar confuses me. But if general consensus is that the sugar makes it more sweet, I'm okay with it :D

Reply:
Yeah, they aren't needed and I don't have any preference, either way is fine for me!